_docs-sources/iac/whats-this/modules.md

```diff
@@ -1,11 +1,21 @@
 # What is a Module?
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Est placerat in egestas erat imperdiet sed. Velit dignissim sodales ut eu sem integer vitae. Mattis rhoncus urna neque viverra. Massa placerat duis ultricies lacus sed turpis tincidunt id. Posuere sollicitudin aliquam ultrices sagittis orci. Egestas quis ipsum suspendisse ultrices gravida dictum fusce ut placerat. Tristique senectus et netus et malesuada. Ipsum dolor sit amet consectetur adipiscing elit pellentesque. Enim nunc faucibus a pellentesque sit amet porttitor eget dolor. Eget nunc lobortis mattis aliquam faucibus purus in. Vulputate dignissim suspendisse in est ante in nibh mauris. Morbi non arcu risus quis varius quam. Cras adipiscing enim eu turpis egestas pretium. Donec enim diam vulputate ut. Ut tortor pretium viverra suspendisse potenti nullam. Euismod elementum nisi quis eleifend quam adipiscing vitae proin sagittis. Sed nisi lacus sed viverra tellus. Laoreet id donec ultrices tincidunt arcu non sodales neque.
+Modules are reusable code to deploy and manage one piece of infrastructure and each one is a battle-tested, best-practices definition of a piece of infrastructure, such as a VPC, ECS cluster, or an Auto Scaling Group.
 
-## Whats a Module
+## When do you use a Module?
 
-It's a bunch of code, ya silly!
+Modules are fairly generic building blocks, so you don't typically deploy a single module directly, but rather, you write code that combines the modules you need for a specific use case. 
```

Comment:
```suggestion
Modules are fairly generic building blocks, so you don't typically deploy a single module directly. Instead, you write code that combines the modules you need for a specific use case. 
```

Reply:
This is a helpful sentence and something we didn't do a good job of explaining before :+1: